src/components/modals/BackupModal.tsx

```diff
@@ -82,7 +82,6 @@ const getStyles = cacheStyles((theme: Theme) => ({
     alignItems: 'center',
     flexDirection: 'column',
     justifyContent: 'space-between',
-    marginTop: theme.rem(2),
     height: '100%'
```

Comment:
I think the `height: 100%` might be the problem. I think we mean `flexGrow: 1`, so the area expands to fill the modal? Explicit dimensions are a big red flag for layout problems, since they erase the intrinsic content size.

Nevertheless, if it works, we can merge.

Reply:
flexGrow causes the image to go away. The best looking version, by just tweaking these styles, is to just remove marginTop